icon_validator/workflow/model.py

```diff
@@ -0,0 +1,152 @@
+from typing import List, Optional, Type, Union
+from dataclasses import dataclass, field
+
+
+class Templates:
+    """
+    Templates is a model thats used for managing the template development lifecycle
+    """
+
+    # TODO Create template struct
```

Comment:
Why is this included here? Is it used? (I'm also not familiar with what a Template is so might be obvious)

Reply:
Oh this was existing code that I pulled over. Ill. get this cleaned up